packages/ui/fixtures/dashboard.tsx

```diff
@@ -0,0 +1,241 @@
+import React from 'react';
+import { Tag, Badge } from 'antd';
+
+export const dashboardColumns = [
+  {
+    title: 'Task Name',
+    dataIndex: 'name',
+    // render: (text) => <a>{text}</a>,
+    render: (value, index) => {
+      if (index > 0) {
+        return <p>{value}</p>;
+      }
+
+      return {
+        children: <p>{value}</p>,
+        // props: {
+        //   colSpan: 7,
+        // },
+      };
+    },
+  },
+  {
+    title: 'Status',
+    render: (text) => {
+      let color = '#d5d5d5';
+      if (text === 'In progress') {
+        color = 'rgba(246 ,140 ,49,1)';
+      }
+      if (text === 'Completed') {
+        color = 'rgba(90 ,170 ,45,1)';
+      }
+
+      return (
+        <div className="flex-row-center">
+          <Tag color={color}>{text}</Tag>
+        </div>
+      );
+    },
+    dataIndex: 'status',
+    align: 'right',
+  },
+  {
+    title: 'Owner',
+    dataIndex: 'owner',
+    render: (text) => {
+      if (text !== '') {
```

Comment:
Nit: Prefer early exit:  `if (!text) return null`

Reply:
we will remove antd, so will change accordingly :)